ci-operator/step-registry/openstack/conf/exportcindercredentials/openstack-conf-exportcindercredentials-commands.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+yq -r '.clouds."'"${OS_CLOUD}"'" | paths(scalars) as $p | "export OS_" + ($p|last|ascii_upcase) + "=\"" + (getpath($p)|tostring) + "\""' \
```

Comment:
Let's not use this one-liner, it's not safe regarding special chars in the password, and potentially has other side effects, such as unexpected results when the clouds.yaml has `regions` entries: https://docs.openstack.org/openstacksdk/latest/user/config/configuration.html#per-region-settings

It's also harder to extend we we need to.

Please go with the python implementation we have battle tested in our CI:
https://github.com/openshift/release/blob/6ce8829103fd9a44baea85bd3fcc951a6c46c74f/ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml#L192-L237